Without this change it fails to compile, giving the following error:

zmalloc.c:55:28: error: missing binary operator before token "("
 #define PREFIX_SIZE (sizeof(size_t))
                            ^
zmalloc.c:59:5: note: in expansion of macro 'PREFIX_SIZE'
 #if PREFIX_SIZE > 0

--- a/src/zmalloc.h
+++ b/src/zmalloc.h
@@ -63,12 +63,10 @@
 
 #ifndef ZMALLOC_LIB
 #define ZMALLOC_LIB "libc"
-#ifdef __GLIBC__
 #include <malloc.h>
 #define HAVE_MALLOC_SIZE 1
 #define zmalloc_size(p) malloc_usable_size(p)
 #endif
-#endif
 
 /* We can enable the Redis defrag capabilities only if we are using Jemalloc
  * and the version used is our special version modified for Redis having
